### ISSUE TITLE
Abnormal CD4+ T cell subsets

### ISSUE THREAD

**Issue by adamklocperk** [obophenotype/human-phenotype-ontology#11264]
**Preferred term label:**
Abnormal CD4+ T cell subsets

**Synonyms**

**Definition (free text, please give PubMed ID)**
Abnormal increase or decrease of any CD4+ T cell subpopulation, measured as percentage of total CD4+ T cells in the blood, compared to a reference range for a given sex and age-group.

**Parent term (use hpo.jax.org/app)**
Abnormal T cell subset distribution HP:0025540

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-1526-4557

**Comment by pnrobinson** [comment]
@adamklocperk I do not understand the suggestion. 
Should we add "Abnormal CD4+ T cell subsets" as a synonym to [Abnormal T cell subset distribution HP:0025540](https://hpo.jax.org/browse/term/HP:0025540)? 
I think it is good to keep the word proportion in the title, because otherwise non-immunologists might think that the particilar cell was abnormal (e.g., too big), rather than that the proportions are changed.

**Comment by adamklocperk** [comment]
Please see my response here: https://github.com/obophenotype/human-phenotype-ontology/issues/11265#issuecomment-3241289346

**Comment by pnrobinson** [comment]
will work on #11265
